src/components/cars/carslist.js

```diff
@@ -1,59 +1,21 @@
 import React, { Component } from 'react';
```

Comment:
what is the difference between cars.js and carlist.js?
why do they both describe same class?